docs/mobile-apps/automated-testing/appium/appium-2-deque-drivers.md

```diff
@@ -0,0 +1,174 @@
+sidebar_label: Appium Deque axe DevTools Integration
+description: Learn how to run accessibility tests using axe DevTools® Mobile Analyzer drivers hosted in Sauce Labs' private cloud.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<p><span className="sauceGreen">Real Devices</span></p>
+
+[Deque's axe™](https://www.deque.com/axe/) is a leading digital accessibility toolkit. 
+The [axe DevTools® for Mobile SDK](https://docs.deque.com/devtools-mobile/appium) extends this functionality to mobile platforms, 
+allowing you to integrate automated accessibility testing directly into your Appium workflows. 
+With this SDK, you can scan mobile app content, identify accessibility issues, and improve compliance with accessibility standards.
+
+The following guide explains how to set up the Sauce Labs integration with [axe DevTools® for Mobile](https://docs.deque.com/devtools-mobile/appium). 
+This integration enables you to run mobile accessibility tests securely and efficiently using Sauce Labs' cloud infrastructure.
+
+## What You'll Need
+
+- A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up))
+- Your Sauce Labs [Username and Access Key](https://app.saucelabs.com/user-settings)
+- Familiarity writing and running [Appium tests](/mobile-apps/automated-testing/appium/)
+- [A Deque Account](https://axe.deque.com/plans) (axe DevTools® for Mobile Product is required) 
+
+
+## Appium 2 Version: `appium2-deque-accessibility`
+
+Sauce Labs offers various versions of Appium 2 in our cloud. One of these is the `appium2-deque-accessibility` version, 
+designed to support accessibility testing on mobile apps for both iOS and Android. This version integrates Appium drivers developed by 
+the Deque team, enabling seamless integration of accessibility scans into your existing test automation workflows.
+
+You can find more details on Sauce Labs' [Appium versions documentation](/mobile-apps/automated-testing/appium/appium-versions/#appium-2x).
+
+## Key Features of `appium2-deque-accessibility`
+
+- ****Includes Deque Drivers :**** This version incorporates the following accessibility drivers provided by Deque:
+    - [axe-appium-uiautomator2-driver](https://docs.deque.com/devtools-mobile/2024.9.18/en/appium-setup#configure-your-tests): Automation name is `AxeUIAutomator2`
+    - [axe-appium-xcuitest-driver](https://docs.deque.com/devtools-mobile/2024.9.18/en/appium-setup#configure-your-tests): Automation name is `AxeXCUITEST`
+:::info Min iOS Version for `axe-appium-xcuitest-driver`:
+- You can use the `axe-appium-xcuitest-driver` only for devices with iOS 17 or above.
+  :::
+- ****Backward Compatibility with Deprecated Plugin:**** While Deque’s  [axeDevToolsMobile Appium Plugin](https://docs.deque.com/devtools-mobile/2024.2.14/en/june-2024-3) 
```

Comment:
I think we don't need this here. We could  add this in a dedicated migration guide to move from the old image (new MR I think).

Reply:
I see you added a migration guide below :)

Reply:
Its not really a feature though? Looking at the section in the preview Its adding a lot of info that is not really a feature?

Reply:
it's not a feature, just hosting the plugin a bit more providing users enough time to switching to use the drivers